### PR TITLE
Racon: Remove python and mummer run deps

### PR DESCRIPTION
**Pull request by ar0ch**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

This is an update to the run dependencies for racon.  The current recipe prevents installs on py3 environments because of a conflicting numpy/matplotlib requirement in the example script.  I don't think these should be enforced as strict run dependencies since the example script is discarded during install.  Also, the Unicycler recipe needs racon and is only py3 compatible

### CODE REVIEW

File: recipes/racon/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 
 build:
   number: 0
-  skip: True # [not py27]
+  skip: False
```
Comment (peterjc) [inline recipes/racon/meta.yaml]: Can you just delete the skip line now that is not used?

Comment (ar0ch) [reply recipes/racon/meta.yaml]: Oh, yeah that would be the right thing to do. Thanks

**Comment by ar0ch** [comment]
@druvus Are we good to merge this change? We're going to do an offline build of the new Unicycler which will pull this as a runtime dependency

Thanks!

**Comment by druvus** [comment]
@ar0ch @peterjc sorry I missed this PR. I have added some modification and agree with to remove python as a requirement.

**Comment by bgruening** [comment]
Thanks all!

**Comment by nekrut** [comment]
somehow this is still in conflict with unicycler because racon 0.5.0 build 1 has this:
```
nsatisfiableError: The following specifications were found to be in conflict:
  - libcurl -> zlib[version='>=1.2.11,<1.3.0a0']
```
is there a reason that this particular zlib is required?

**Pull request merged**